README.md

```diff
@@ -105,20 +105,20 @@ Setup Auto-DL:
 [2]: https://www.linkedin.com/company/autodl/
 [3]: https://join.slack.com/t/autodl/shared_invite/zt-qagxiwub-ywRM_oBvvF~F7YNtlBqy_Q
 
-#### Have doubts?
+#### Have doubts? 🤷‍♂️
 
 Read the [FAQ](#todo) or send us a mail at info@auto-dl.tech
 
 For more details, visit our [website](https://auto-dl.tech/)
 
 ---
 
-## Contributors
+## Contributors 🤼
```

Comment:
```suggestion
## Contributors 💯 
```
Well, this or something else but wrestling, we sure don't want our contributors to wrestle 😛